apps/cloudinary/src/index.js

```diff
@@ -162,10 +187,28 @@ setup({
     {
       "id": "startFolder",
       "name": "Starting folder",
-      "description": "Path to starting folder for the media library dialog",
+      "description": "A path to a folder, which the Cloudinary Media Library will automatically browse to on load",
       "type": "Symbol",
       "required": false,
       "default": ""
+    },
+    {
+      "id": "quality",
+      "name": "Media Quality",
+      "description": "The quality level of your assets. This can be a fixed number ranging from 1-100, or you can get Cloudinary to decide the most optimized level by setting to 'auto'. More available options are auto:low/auto:eco/auto:good/auto:best. If you wish to use the original level, set to 'none'",
```

Comment:
```suggestion
      "description": "The quality level of your assets. This can be a fixed number ranging from 1-100, or you can get Cloudinary to decide the most optimized level by setting it to 'auto'. More options are available such as: auto:low/auto:eco/auto:good/auto:best. If you wish to use the original level, set it to 'none'.",
```